tasks/push-snapshot/push-snapshot.yaml

```diff
@@ -111,6 +119,9 @@ spec:
             --format '{{.Digest}}' \
             "docker://${containerImage}" 2>/dev/null)
 
+          RESULTS_JSON=$(jq --arg i "$i" --argjson arches "$arches" --arg name "$name" --arg sha "$origin_digest" \
```

Comment:
If you change `--arg i "$i"` to `--argjson i "$i"`, you don't need the `|tonumber`. At least that's what we've been doing elsewhere. But if you prefer this, it's fine as well - I guess it's hard to argue one is better than the other.

Reply:
Considering `i` is an integer, not a json, I think it is more clear with the `|tonumber`

Reply:
ok